src/devhub/components/templates/AppLayout.jsx

```diff
@@ -1,5 +1,9 @@
+const data = fetch(`https://httpbin.org/headers`);
+const gatewayURL = data?.body?.headers?.Origin ?? "";
+
+// we need fixed positioning for near social and not for org
 const Theme = styled.div`
-  position: fixed;
+  position: ${gatewayURL.includes("near.org") ? "inherit" : "fixed"};
```

Comment:
using an expression inside the`styled.div` leads to that the transaction modal stops working (which the tests detect and then fails). So instead you have to do something like this:

<img width="635" alt="image" src="https://github.com/NEAR-DevHub/neardevhub-bos/assets/9760441/24df62e0-87a8-4ffa-8851-1633aa267b23">

Reply:
I have made the update